src/rebloom.c

```diff
@@ -129,6 +130,15 @@ static int BFReserve_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv,
         return RedisModule_ReplyWithError(ctx, "ERR bad capacity");
     }
 
+    long long expansion = BF_DEFAULT_EXPANSION;
+    int ex_loc = RMUtil_ArgIndex("EXPANSION", argv, argc);    
+    if (ex_loc != -1) {
+        if (RedisModule_StringToLongLong(argv[ex_loc + 1], &expansion) != REDISMODULE_OK) {
+            RedisModule_ReplyWithError(ctx, "ERR bad expansion");
```

Comment:
Should be:
```
return RedisModule_ReplyWithError(ctx, "ERR bad expansion");
```
Contrary to correct intuition, methods should't return REDISMODULE_ERR when faced with an error. Thus, RedisModule_ReplyWithError returns REDISMODULE_OK.

Reply:
@rafie 
You are correct and it is all over the file.
Talking this opportunity to fix this.